packages/amplication-client/src/Components/ResourceSelectorItem.tsx

```diff
@@ -9,12 +11,23 @@ type Props = {
 const CLASS_NAME = "commit-selector-item";
 
 export const ResourceSelectorItem = ({ resource }: Props) => {
+  const createdAtHour = resource
+    ? formatTimeToNow(new Date(resource?.createdAt))
+    : null;
+
+  const truncateServiceName = truncateWithEllipsis(
+    resource?.name.trim(),
+    15,
+    "Service Name"
+  );
```

Comment:
you should use useMemo for things like this otherwise the function will be called with every render